setup.cfg

```diff
@@ -51,39 +50,41 @@ install_requires =
 [options.extras_require]
 # The list of available extras is also in the installation docs, if you add or remove one please edit it there as well.
 asdf =
-  asdf-astropy>=0.1.1
-  asdf>=2.8.0
+  asdf-astropy>=0.2.0
+  asdf>=2.11.0
 dask =
-  dask[array]>=2021.4.0
+  dask[array]>=2022.5.2
 image =
-  scikit-image>=0.18.0
+  scikit-image>=0.19.0
   # !=1.10.0 due to https://github.com/scipy/scipy/issues/17718
-  scipy>=1.7.0,!=1.10.0
+  scipy>=1.8.0,!=1.10.0
 jpeg2000 =
   glymur>=0.9.1,!=0.9.5
-  lxml>=4.8.0,!=5.0.0
+  lxml>=4.9.0,!=5.0.0
 map =
   matplotlib>=3.5.0
   mpl-animators>=1.0.0
-  reproject
+  reproject>=0.9.0
   # !=1.10.0 due to https://github.com/scipy/scipy/issues/17718
-  scipy>=1.7.0,!=1.10.0
+  scipy>=1.8.0,!=1.10.0
+opencv =
```

Comment:
Worth adding a changelog entry for this new extra?

Reply:
Sure, it isn't installed by default, so people shouldn't notice a difference, it's just for the sake of our setup.cfg.